### PR TITLE
openshift_ovirt: fix inventory items creation

### PR DESCRIPTION
**Pull request by rgolangh**

### CODE REVIEW

File: roles/openshift_ovirt/tasks/create_inventory_from_vms.yaml [hunk 1]
```diff
@@ -11,7 +14,7 @@
 - name: Add compute Vms to the nodes inventory group
   add_host:
     groups: nodes
```
Comment (richm) [inline roles/openshift_ovirt/tasks/create_inventory_from_vms.yaml]: should this also be `[nodes]`?

Comment (sdodson) [reply roles/openshift_ovirt/tasks/create_inventory_from_vms.yaml]: No, I think that's correct.

File: roles/openshift_ovirt/tasks/create_inventory_from_vms.yaml [hunk 2]
```diff
@@ -20,23 +23,23 @@
...
     openshift_schedulable: true
   with_items: "{{ openshift_ovirt_vm_manifest }}"
   when: item.count > 0 and item.name == 'infra'
 
 - name: Add etcd Vms to the etcd inventory group
   add_host:
-    hostname: '{{ [item.name, item.count - 1, ".", openshift_ovirt_dns_zone] | join }}'
-    groups: 'etcd'
+    hostname: '{{ (item.name, (item.count > 1) | ternary((("[0:",item.count - 1 , "]") | join), "0"), ".", openshift_ovirt_dns_zone) | join }}'
+    groups: [etcd]
```
Comment (sdodson) [inline roles/openshift_ovirt/tasks/create_inventory_from_vms.yaml]: Hmm, per the earlier comment I think these should be uniformly just the string.

File: roles/openshift_ovirt/tasks/create_inventory_from_vms.yaml [hunk 3]
```diff
@@ -20,23 +23,23 @@
...
   with_items: "{{ openshift_ovirt_vm_manifest }}"
   when: item.count > 0 and item.name == 'etcd'
 
 - name: Add load balancers Vms to the lb inventory group
   add_host:
-    groups: 'lb'
-    hostname: '{{ [item.name, item.count - 1, ".", openshift_ovirt_dns_zone] | join }}'
+    groups: [lb]
```
Comment (sdodson) [inline roles/openshift_ovirt/tasks/create_inventory_from_vms.yaml]: here too.

Comment (rgolangh) [reply roles/openshift_ovirt/tasks/create_inventory_from_vms.yaml]: done

File: roles/openshift_ovirt/tasks/create_inventory_from_vms.yaml (hunk 2 above)
Comment (rgolangh) [reply roles/openshift_ovirt/tasks/create_inventory_from_vms.yaml]: done

**Pull request merged**